python/tests/conftest.py

```diff
@@ -117,6 +125,38 @@ def moto_s3_uri_incl_bucket(moto_s3_endpoint_and_credentials):
     ] + ":" + bucket + "?access=" + aws_access_key + "&secret=" + aws_secret_key + "&port=" + port
 
 
+@pytest.fixture
+def moto_azure_endpoint_and_credentials(azurite_port, spawn_azurite):
```

Comment:
Note `moto` is the library that emulates AWS S3 (like what Azurite does for Azure), so this name makes no sense.